### DISCUSSION TITLE
Facets txtF_mv

### DISCUSSION THREAD

**Discussion by ChinuxMind** [vufind-org/vufind#3880] [Q&A]
Hi everyone,
I´ve a problem with the facets, 
They don't appear on the main page, in the facets.ini file
[Results]
ubicacion_txtF_mv = Ubicación
collapsedFacets = fondo_txtF_mv,tematica_txtF_mv,ubicacion_txtF_mv,dewey-hundreds
[Advanced]
ubicacion_txtF_mv = Ubicación
[HomePage]
ubicacion_txtF_mv = Ubicación

searches.ini
[Basic_Searches]
ubicacion_txtF_mv = Ubicación

File marc_local.properties in folder import 

ubicacion_txtF_mv = 930a

I'have DRIVER Aleph and harvest through OAI-PMH
Vufind 9
Regards!!

**Comment by demiankatz** [comment]
@ChinuxMind, is the custom facet failing to appear anywhere, or is it only missing on the home page?

If it's only missing on the home page, you may need to clear your cache (the contents of the `$VUFIND_LOCAL_DIR/cache` directory) so that it will appear.

If it's missing everywhere, do you see values in the custom field if you query Solr directly? (e.g. go to the Solr admin panel through the web and query for `ubicacion_txtF_mv:*` to see if any results come back). If you're not seeing any values, is it possible that you didn't reindex your records after customizing your marc_local.properties?

Please let me know if you need more help!

**Comment by ChinuxMind** [comment]
Good morning
Following your instructions, when I run the reindex, I managed to see that it goes to  /usr/local/vufind/import/marc_local. properties nstead of  /usr/local/vufind/local/import/ marc_local. properties
***@***.***
It works but not as good route practices local/vufind/local ***@***.***

Privilegios
***@***.***

***@***.***

I changed the permissions of /usr/local/vufind/local/import/marc_local.properties, but it didn't work either
***@***.***
File reindex
***@***.***
#!/bin/sh

# Load VuFind profile
. /etc/profile.d/vufind.sh

export INDEX_OPTIONS='-Xms1024m -Xmx1024m -DentityExpansionLimit=0'

SCRIPTDIR=/usr/local/vufind/harvest
WORKDIR=/usr/local/vufind/local/harvest
HARVEST=iteso
LOCKDIR="$WORKDIR/$HARVEST/cron-harvest.lock"
cd $WORKDIR

echo "Attempting to create lock directory..."
if mkdir $LOCKDIR 2>/dev/null; then
    echo "Lock directory created."



    mv $WORKDIR/$HARVEST/processed/*.delete $WORKDIR/$HARVEST/

    $SCRIPTDIR/batch-delete.sh $HARVEST


    mv $WORKDIR/$HARVEST/processed/*.xml $WORKDIR/$HARVEST/

    $SCRIPTDIR/batch-import-marc.sh $HARVEST

    rm -f $WORKDIR/harvest.lock

    exit 0

else

    echo "Failed to acquire lock!  lockfile(1) returned $?"
    exit 1
fi

Regards
Abraham Arenivas

De: Demian Katz ***@***.***>
Enviado el: sábado, 24 de agosto de 2024 05:36 a. m.
Para: vufind-org/vufind ***@***.***>
CC: ARENIVAS DIAZ, JOSE ABRAHAM ***@***.***>; Mention ***@***.***>
Asunto: Re: [vufind-org/vufind] Facets txtF_mv (Discussion #3880)


Precaución: Este correo se originó desde fuera de la Institución. No haga clic en enlaces ni abra archivos adjuntos, a menos que reconozca el remitente y tenga conocimiento de que el contenido es seguro.

@ChinuxMind<https://github.com/ChinuxMind>, is the custom facet failing to appear anywhere, or is it only missing on the home page?

If it's only missing on the home page, you may need to clear your cache (the contents of the $VUFIND_LOCAL_DIR/cache directory) so that it will appear.

If it's missing everywhere, do you see values in the custom field if you query Solr directly? (e.g. go to the Solr admin panel through the web and query for ubicacion_txtF_mv:* to see if any results come back). If you're not seeing any values, is it possible that you didn't reindex your records after customizing your marc_local.properties?

Please let me know if you need more help!

-

**Comment by demiankatz** [comment]
Is it possible that the VUFIND_LOCAL_DIR environment variable is not exported in the context where you are running the import? That variable is needed to allow the tools to find your local directory. I wonder if the way you are loading the profile is not working correctly in your script.

- Demian

From: ChinuxMind ***@***.***>
Sent: Wednesday, August 28, 2024 3:25 PM
To: vufind-org/vufind ***@***.***>
Cc: Demian Katz ***@***.***>; Comment ***@***.***>
Subject: [EXTERNAL] Re: [vufind-org/vufind] Facets txtF_mv (Discussion #3880)

Good morning
Following your instructions, when I run the reindex, I managed to see that it goes to /usr/local/vufind/import/marc_local. properties nstead of /usr/local/vufind/local/import/ marc_local. properties
***@***.***<mailto:***@***.***>
It works but not as good route practices local/vufind/local ***@***.***<mailto:***@***.***>

Privilegios
***@***.***<mailto:***@***.***>

***@***.***<mailto:***@***.***>

I changed the permissions of /usr/local/vufind/local/import/marc_local.properties, but it didn't work either
***@***.***<mailto:***@***.***>
File reindex
***@***.***<mailto:***@***.***>
#!/bin/sh

# Load VuFind profile
. /etc/profile.d/vufind.sh

export INDEX_OPTIONS='-Xms1024m -Xmx1024m -DentityExpansionLimit=0'

SCRIPTDIR=/usr/local/vufind/harvest
WORKDIR=/usr/local/vufind/local/harvest
HARVEST=iteso
LOCKDIR="$WORKDIR/$HARVEST/cron-harvest.lock"
cd $WORKDIR

echo "Attempting to create lock directory..."
if mkdir $LOCKDIR 2>/dev/null; then
echo "Lock directory created."



mv $WORKDIR/$HARVEST/processed/*.delete $WORKDIR/$HARVEST/

$SCRIPTDIR/batch-delete.sh $HARVEST


mv $WORKDIR/$HARVEST/processed/*.xml $WORKDIR/$HARVEST/

$SCRIPTDIR/batch-import-marc.sh $HARVEST

rm -f $WORKDIR/harvest.lock

exit 0

else

echo "Failed to acquire lock! lockfile(1) returned $?"
exit 1
fi

Regards
Abraham Arenivas

De: Demian Katz ***@***.***<mailto:***@***.***>>
Enviado el: sábado, 24 de agosto de 2024 05:36 a. m.
Para: vufind-org/vufind ***@***.***<mailto:***@***.***>>
CC: ARENIVAS DIAZ, JOSE ABRAHAM ***@***.***<mailto:***@***.***>>; Mention ***@***.***<mailto:***@***.***>>
Asunto: Re: [vufind-org/vufind] Facets txtF_mv (Discussion #3880)


Precaución: Este correo se originó desde fuera de la Institución. No haga clic en enlaces ni abra archivos adjuntos, a menos que reconozca el remitente y tenga conocimiento de que el contenido es seguro.

@ChinuxMind<https://github.com/ChinuxMind>, is the custom facet failing to appear anywhere, or is it only missing on the home page?

If it's only missing on the home page, you may need to clear your cache (the contents of the $VUFIND_LOCAL_DIR/cache directory) so that it will appear.

If it's missing everywhere, do you see values in the custom field if you query Solr directly? (e.g. go to the Solr admin panel through the web and query for ubicacion_txtF_mv:* to see if any results come back). If you're not seeing any values, is it possible that you didn't reindex your records after customizing your marc_local.properties?

Please let me know if you need more help!

-
Reply to this email directly, view it on GitHub<https://github.com/vufind-org/vufind/discussions/3880#discussioncomment-10437350>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AY4OZR7BXAZCMZZ2OGEW5T3ZTBV2VAVCNFSM6AAAAABNA46CZKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBTG4ZTKMA>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***<mailto:***@***.******@***.***>>>


-
